pkg/objstore/swift/swift.go

```diff
@@ -58,13 +69,57 @@ func NewContainer(logger log.Logger, conf []byte) (*Container, error) {
 		Password:         sc.Password,
 		DomainID:         sc.DomainId,
 		DomainName:       sc.DomainName,
-		TenantID:         sc.TenantID,
-		TenantName:       sc.TenantName,
+		TenantID:         sc.ProjectID,
+		TenantName:       sc.ProjectName,
 
 		// Allow Gophercloud to re-authenticate automatically.
 		AllowReauth: true,
 	}
 
+	// The term `tenant` is used in the deprecated OpenStack Identity v2.
+	// In Identity v3 the term `project` is used instead.
+	switch {
+	case sc.TenantName != "":
+		level.Warn(logger).Log("msg", "usage of tenant_name is deprecated. use project_name instead")
+		authOpts.TenantName = sc.TenantName
+	case sc.TenantID != "":
+		level.Warn(logger).Log("msg", "usage of tenant_id is deprecated. use project_id instead")
+		authOpts.TenantID = sc.TenantID
+	case sc.ProjectName != "" || sc.ProjectID != "":
+		level.Info(logger).Log("msg", "using project_id or project_name")
```

Comment:
I think we should remove this, it isn't a good message and IMO doesn't add any value

Reply:
right. removed